tests/src/Endpoint/IdentifierEndpointTest.php

```diff
@@ -5,42 +5,35 @@
 namespace OpenEuropa\Tests\CdtClient\Endpoint;
 
 use GuzzleHttp\Psr7\Response;
+use OpenEuropa\CdtClient\Endpoint\IdentifierEndpoint;
 use OpenEuropa\CdtClient\Exception\ValidationErrorsException;
-use OpenEuropa\CdtClient\Model\Response\Token;
 use OpenEuropa\CdtClient\Model\Response\ValidationErrors;
 use OpenEuropa\Tests\CdtClient\Traits\AssertTestRequestTrait;
-use OpenEuropa\Tests\CdtClient\Traits\ClientTestTrait;
+use OpenEuropa\Tests\CdtClient\Traits\ApiTestTrait;
 use PHPUnit\Framework\TestCase;
 
 /**
  * @coversDefaultClass \OpenEuropa\CdtClient\Endpoint\IdentifierEndpoint
  */
 class IdentifierEndpointTest extends TestCase
 {
-    use ClientTestTrait;
+    use ApiTestTrait;
     use AssertTestRequestTrait;
 
     /**
      * @dataProvider providerTestIdentifier
      *
      * @param array<string, mixed> $clientConfig
      * @param Response[] $responses
-     * @param mixed $expectedResult
      *
      * @covers \OpenEuropa\CdtClient\Endpoint\IdentifierEndpoint
      * @covers \OpenEuropa\CdtClient\Endpoint\EndpointBase
+     * @covers \OpenEuropa\CdtClient\Endpoint\AuthorizedEndpointBase
      * @covers \OpenEuropa\CdtClient\Http\Rest
      */
     public function testIdentifier(string $correlationId, array $clientConfig, array $responses, mixed $expectedResult): void
     {
-        $token = (new Token())->setAccessToken('JWT_TOKEN')
```

Comment:
I took a microservice-centric approach here. Instead of creating the client and modifying its reflection to access the endpoints illegally, I've just called the constructor of the endpoint. This is shorter and clearer, and makes the test more isolated.

The missing lines are now replaced by `ApiClientTest` and `ApiFactoryTest`.